src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/CSharpIdentifier.cs

```diff
@@ -1,20 +1,28 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Globalization;
 using System.Text;
 
 namespace Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X
 {
     internal static class CSharpIdentifier
     {
+        private const string CshtmlExtension = ".cshtml";
```

Comment:
🆙 📅

Reply:
Do we want to strip out any extension or cshtml extension?

Reply:
Can we just do `Path.GetFileNameWithoutExtension`?

Reply:
That wouldn't work because it trims out the remainder of the path:

```
> [IO.Path]::GetFileNameWithoutExtension('/work/razor/Index.cshtml')
Index
``

I can look for the last "." and trim everything past it.

Reply:
Bump

Reply:
OK well this seems fine for now then.